src/util/dwarf.rs

```diff
@@ -150,18 +150,34 @@ impl FundType {
             FundType::Vec2x32Float => "__vec2x32float__",
         })
     }
+
+    pub fn parse_int(value: u16) -> Result<Self, TryFromPrimitiveError<Self>> {
+        if value >> 8 == 0x1 {
+            // Can appear in erased tags
+            Self::try_from(value & 0xFF)
```

Comment:
Interesting, can the user types not appear here?

Reply:
User types can still appear here, this is a hack to parse these values:
```
FundType::parse_int: 0101
FundType::parse_int: 0103
FundType::parse_int: 0105
FundType::parse_int: 0106
FundType::parse_int: 0107
FundType::parse_int: 0108
FundType::parse_int: 0109
FundType::parse_int: 010E
FundType::parse_int: 010F
```
maybe it would be more robust to add enums for all these explicitly, like `MwSignedShort = 0x0105`?